#! /bin/sh /usr/share/dpatch/dpatch-run
## 123-extern-memmove-only-if-not-macro.dpatch by  <pertusus@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Declare memmove() only if it is not already defined as a macro.

@DPATCH@
diff -urNad cernlib-2006.dfsg.2~/src/packlib/kuip/kuip/kstring.h cernlib-2006.dfsg.2/src/packlib/kuip/kuip/kstring.h
--- cernlib-2006.dfsg.2~/src/packlib/kuip/kuip/kstring.h	2007-04-21 19:21:26.000000000 -0400
+++ cernlib-2006.dfsg.2/src/packlib/kuip/kuip/kstring.h	2007-04-21 19:22:20.000000000 -0400
@@ -21,8 +21,10 @@
  */
 
 #ifndef WIN32
+#ifndef memmove
 extern void* memmove( void* dst, const void* src, size_t n );
 #endif
+#endif
 
 /* GF. make conform to kkern.c#if !defined(__convexc__)*/
 #ifndef HAVE_STRCASECMP
